concepts/sepsis/sepsis3.sql

```diff
@@ -0,0 +1,57 @@
+WITH s1 as 
+(
+  SELECT 
+    sofa.* 
+    , soi.subject_id
+    , soi.ab_id
+    , soi.antibiotic
+    , soi.antibiotic_time
+    , soi.culture_time
+    , soi.suspected_infection
+    , soi.suspected_infection_time
+    , soi.specimen
+    , soi.positive_culture
+  FROM `lcp-internal.sepsis_test.suspicion_of_infection` as soi
+  INNER JOIN `lcp-internal.sepsis_test.sofa` as sofa
+    ON soi.stay_id = sofa.stay_id 
+    AND sofa.starttime >= DATETIME_SUB(soi.suspected_infection_time, INTERVAL 48 HOUR)
+    AND sofa.endtime <= DATETIME_ADD(soi.suspected_infection_time, INTERVAL 24 HOUR)
```

Comment:
you can probably just join with sofa.endtime. most of the time, this is when the measurement was documented, so it will be the more accurate time.

Reply:
I thought sofa.endtime is a value that we compute? (e.g. isn't it just the end of the hour?)

Reply:
It is - the main thing is `endtime` is where the equality is when extracting data for SOFA, e.g. here: https://github.com/MIT-LCP/mimic-iv/blob/sepsis3-related/concepts/score/sofa.sql#L104-L111

You can see we do `charttime > starttime AND charttime <= endtime` - since most measurements are documented on the hour (10:00, 11:00, etc), most measurements will occur at the sofa `endtime`. So I'd just use `endtime`.